src/pretix/static/pretixpresale/js/ui/main.js

```diff
@@ -82,7 +82,13 @@ var form_handlers = function (el) {
                     Math.abs(+new Date(opts.minDate) - new Date()) < Math.abs(+new Date(opts.maxDate) - new Date())
             ) ? opts.minDate : opts.maxDate;
         }
-        $(this).datetimepicker(opts);
+        $(this).datetimepicker(opts).on("dp.hide", function() {
+            var $dtp = $(this).data("DateTimePicker");
+            var currentViewDate = $dtp.viewDate();
+            window.setTimeout(function () {
+                $dtp.viewDate(currentViewDate);
+            }, 500);
```

Comment:
I am quite capable of still triggering the bug by clicking faster than 500ms, so I suggest 50ms. (Note that I tested with the bootstrap-datetimepicker.js change reverted, otherwise the issue does not occur.)

```suggestion
            }, 50);
```

Reply:
Well, you have fast fingers. :) I changed it to 50ms, not sure why I chose 500 in the first place.